integration/query_test.go

```diff
@@ -1179,15 +1190,48 @@ func TestQueryCommandGetMore(t *testing.T) {
 				Message: "BSON field 'getMore.collection' is missing but a required field",
 			},
 		},
+		"UnsetAllBatchSize": {
```

Comment:
These tests address this comment https://github.com/FerretDB/FerretDB/pull/2864#discussion_r1243195284